cmd/cli/install.go

```diff
@@ -229,6 +231,14 @@ func (i *installCmd) loadOSMChart() error {
 
 func (i *installCmd) resolveValues() (map[string]interface{}, error) {
 	finalValues := map[string]interface{}{}
+
```

Comment:
Can we validate that the input is in the correct format? Or does strvals do that for us (and return a good error message)?

Reply:
`strvals` will do that. It gives reasonable errors, but I wrapped the ones here with a hint that they came from `--set`. I also added a unit test that shows an example error.